crates/cargo-contract/src/util/tests.rs

```diff
@@ -24,6 +24,7 @@ use anyhow::{
 };
 use std::{
     fs,
+    io::Write,
```

Comment:
Can you address this warning?

Reply:
where the warning coming from
I can't reproduce it with `cargo build`, `cargo check` nor `cargo test`

Reply:
You'll need to reproduce it using the CI (I assume you're not using a Windows machine for development). You can comment out the other jobs in the `ci.yml` file in order to make your test cycle a bit quicker. I can also help you debug in the next couple of days

Reply:
I got a VM to sort out these windows issues.
The warning is gone, as well as the previous CI error. Now I'm stuck in another one, but I'll figure it out soon.